docs/architecture/stage_list.md

```diff
@@ -0,0 +1,1060 @@
+---
+---
+
+Stage list
+----------
```

Comment:
It is not incorrect, just incomplete and unreadable. I can make it not visible, but it is not like this is not exactly what is on the site currently and absence thereof would be better state of affairs (IMO of course).